src/toil/batchSystems/mesos/batchSystem.py

```diff
@@ -509,3 +510,10 @@ def __mbToBytes(mem):
         used when converting Mesos reqs to Toil reqs
         """
         return mem * 1024 * 1024
+
+    @staticmethod
+    def supportsWorkerCleanup():
+        """
+        Indicates whether the batch system supports the cleanup of workers on shutdown
```

Comment:
No need to duplicate the overridden method's docstring (unless you promise to keep these copies consistent from now until all eternity).